docs/pages/v3/governance.md

```diff
@@ -56,6 +56,14 @@ Each time an immutable parameter is set via governance proposal, a new Comet imp
 
 To set specific protocol parameters in a proposal, the Timelock must call all of the relevant set methods on the *Configurator* contract, followed by `deployAndUpgradeTo` on the *CometProxyAdmin* contract.
 
+## Governance on Other EVM Chains
+
+The Compound III protocol can be deployed on any EVM chain. The deployment must have access to on-chain asset prices and messages passed from Ethereum Mainnet. The [Timelock](/v2/governance/#timelock) on Mainnet is the administrator of all community sanctioned instances of Compound III.
+
+Each deployment outside of Mainnet needs to have a [Bridge Receiver](https://github.com/compound-finance/comet/blob/main/contracts/bridges/BaseBridgeReceiver.sol) and Local Timelock contract on its chain. Governance proposals executed on Ethereum must be read by the chain's bridge and published to the Bridge Receiver. Local Timelocks have an additional delay of one day before Comet admin functions can be called.
```

Comment:
> Local Timelocks have an additional delay of one day

The delay is configurable, so it's not necessarily one day. It's whatever value was set at contract creation